ted/src/main/java/frc/robot/control/AutonomousChooser.java

```diff
@@ -19,27 +19,36 @@
 import com.pathplanner.lib.util.PIDConstants;
 import com.pathplanner.lib.util.ReplanningConfig;
 
+import frc.robot.commands.AutoShooterSpinUpCommand;
 import frc.robot.commands.ButtonPressCommand;
 import frc.robot.commands.IntakeAndFeedNoteCommand;
-import frc.robot.commands.ShooterShootCommand;
 import frc.robot.commands.RemoveNoteCommand;
+import frc.robot.commands.ShooterSetAngleCommand;
+import frc.robot.commands.ShooterSetAngleWithVisionOneShotCommand;
 import frc.robot.common.FeederMode;
-import frc.robot.control.SubsystemCollection;
+import frc.robot.commands.FeederLaunchNote;
 
+/**
+* a class for choosing different auto modes from shuffleboard
+*/
 public class AutonomousChooser {
     private SubsystemCollection subsystems;
     private final SendableChooser<AutonomousPath> autonomousPathChooser = new SendableChooser<>();
 
     private Command blue123SourceSide;
     private Command red123SourceSide;
     private Command blueWing;
-    private Command redWing;
+    private Command blueFourNote;
+    private Command blueRush;
```

Comment:
rush auto should not be part of chooser until it's tested. Would be really bad to accidentally run this.

Reply:
remove the rush auto from this file in all the places below, too

Reply:
sigh....

Reply:
removed in all areas in Auto chooser